x-pack/plugins/index_management/__jest__/client_integration/home/indices_tab.test.ts

```diff
@@ -247,20 +270,6 @@ describe('<IndexManagementHome />', () => {
       // The unfreeze action should not be present anymore
       expect(exists('unfreezeIndexMenuButton')).toBe(false);
     });
-  });
-
-  describe('Edit index settings', () => {
```

Comment:
I think we should actually keep this describe here as it refers to the Edit index settings tab from the flyout and not to the index actions

Reply:
Done